config/application.rb

```diff
@@ -56,5 +57,7 @@ class Application < Rails::Application
     config.autoload_paths << Rails.root.join('lib')
 
     config.active_job.queue_adapter = :sidekiq
+
+    config.middleware.insert_before 0, SameSiteSecurity::Middleware
```

Comment:
If you call this `SameSiteSecurityMiddleware` and put it in `lib/same_site_security_middleware.rb`, Rails will be able to autoload it without the `require_relative`